pywbem/_subscription_manager.py

```diff
@@ -990,11 +995,14 @@ def add_subscriptions(self, server_id, filter_path, destination_paths=None,
 
     def get_owned_subscriptions(self, server_id):
         """
-        Return the owned indication subscriptions for a WBEM server that have
-        been created by this subscription manager.
+        Return the indication subscriptions in a WBEM server that are owned by
+        this subscription manager.
 
         This function accesses only the local list of owned subscriptions; it
-        does not contact the WBEM server.
+        does not contact the WBEM server. The local list of owned subscriptions
+        is discovered from the WBEM server when the server is registered with
+        the subscription manager, and is from then on maintained as changes
```

Comment:
Another case "is from then on maintained" to "is maintained from then on"

Reply:
DONE